### PR TITLE
Adding Janette's first class reflection

### PR DESCRIPTION
**Pull request by luznette**
Ready to go! 👍

### CODE REVIEW

File: _posts/luznette/2020-10-07-class-reflection-luznette.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+thisShape = instructions()
+draw(thisShape)
+ask = input("Would you like to draw again? (Yes or No):")
+while ask=="yes" or ask=="Yes":
+  thisShape = instructions()
+  draw(thisShape)
+  ask = input("Would you like to draw again? (Yes or No):")
+else:
+  print ("Thank you!")
+  ```
```
Comment (MishaCienfuegos) [inline _posts/luznette/2020-10-07-class-reflection-luznette.md]: This is a great place to use Markdown to format this section as code! In order for it to show up correctly I think you will want to change the syntax just a bit. [Specify the language](https://www.markdownguide.org/extended-syntax/#syntax-highlighting) (python) on the first line and then put your first line of code on a new line, like so:
```
```python
print(your-code)
```

**Review by MishaCienfuegos — CHANGES_REQUESTED** [review]
This looks great!

**Comment by luznette** [comment]
Hi @MishaCienfuegos ! I think I made the necessary changes :)

**Review by MishaCienfuegos — APPROVED** [review]
Awesome, this looks great!

**Pull request merged**